### PR TITLE
CLI args for easier simulated games

### PR DESCRIPTION
**Pull request by MathewMacDougall**
### Description

* Fixed some TODOS (from me last year :grin: ) where certain UI components had hardcoded default values that didn't read from the parameters
* Added some CLI args to make it easier to start simulated games. The script now automatically sets defending side, channel, and team color

### Testing Done

Manual testing

### Resolved Issues

None I'm aware of

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/shared/parameter/config_definitions/full_system_main_command_line_args.yaml [hunk 1]
```diff
@@ -37,3 +37,30 @@
     description: >-
         The directory to output logs to. Absolute paths are recommended as the working directory
         is inside the bazel-out directory.
+
+- int:
```
Comment (jonathanlew) [inline src/shared/parameter/config_definitions/full_system_main_command_line_args.yaml:44]: nit
```suggestion
- uint:
```

File: src/shared/parameter/config_definitions/ai_control_config.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@
         the last commands they were given. DO NOT USE in place of the e-stop.
 - bool:
     name: override_ai_play
-    value: true
+    value: False
```
Comment (jonathanlew) [inline src/shared/parameter/config_definitions/ai_control_config.yaml]: nit: lower case

**Review by jonathanlew — COMMENTED** [review]
very nice

**Review by kelvinkoon — COMMENTED** [review]
Wise alumni blessing us with a PR 🙏 
LGTM besides Jon's comments :)

**Comment by MathewMacDougall** [comment]
http://gph.is/IOzXH4

File: src/shared/parameter/config_definitions/ai_control_config.yaml (hunk 2 above)
Comment (MathewMacDougall) [reply src/shared/parameter/config_definitions/ai_control_config.yaml]: done

File: src/shared/parameter/config_definitions/full_system_main_command_line_args.yaml (hunk 1 above)
Comment (MathewMacDougall) [reply src/shared/parameter/config_definitions/full_system_main_command_line_args.yaml:44]: Tried this but it seems using `uint` params causes a strange compilation issue, so there might be an issue with DynamicParams generation. Made a ticket to investigate: https://github.com/UBC-Thunderbots/Software/issues/2132

**Review by jonathanlew — APPROVED** [review]
:100:

**Review by amrm3lm — APPROVED** [review]
lgtm

**Pull request merged**